judge/forms.py

```diff
@@ -24,7 +25,7 @@
     TOTP_CODE_LENGTH: {
         'regex_validator': RegexValidator(
             f'^[0-9]{{{TOTP_CODE_LENGTH}}}$',
-            _(f'Two-factor authentication tokens must be {TOTP_CODE_LENGTH} decimal digits.'),
+            format_lazy(_('Two-factor authentication tokens must be {count} decimal digits.'), count=TOTP_CODE_LENGTH),
```

Comment:
While you are at it, why not use `ngettext`? Imagine changing the code length to 8 and then having to do eclipsis when translating to Irish...

Reply:
Done